src/TimesheetGPT.WebUI/Pages/GraphChat.razor

```diff
@@ -0,0 +1,112 @@
+@page "/chat"
+
+@inject GraphServiceClient GraphServiceClient
+@inject MicrosoftIdentityConsentAndConditionalAccessHandler ConsentHandler
+@inject TimesheetService TimesheetService
+
+<MudContainer MaxWidth="MaxWidth.Medium" Style="padding-bottom: 2rem;">
+    <MudText Typo="Typo.h1">Chat with GraphAPI</MudText>
+    <MudText Typo="Typo.subtitle1">
+        Hi @_name, ask questions about sent emails, meetings attended, and more.
+    </MudText>
+    <MudCard>
+        <MudCardContent>
+            @foreach (var message in _messages)
+            {
+                <div class="chat-bubble @(message.Sender == Sender.User ? "user" : "")">@message.MessageText</div>
+            }
+        </MudCardContent>
+        <MudCardActions>
+            <MudTextField @bind-Value="@_userInput" Placeholder="Type your message..." FullWidth Variant="Variant.Filled"/>
+            <MudButton Variant="Variant.Filled" Color="Color.Primary" OnClick="SendMessage">Send</MudButton>
+        </MudCardActions>
+    </MudCard>
+</MudContainer>
+
+<style>
+    .chat-bubble {
+        padding: 10px;
+        margin: 5px;
+        background-color: #f1f1f1;
+        border-radius: 5px;
+    }
+    
+    .chat-bubble.user {
+        background-color: #ddd;
+        text-align: right;
+    }
+</style>
```

Comment:
don't add styles like this, use scoped css (if it gets trimed look at the :deep css selector)

Reply:
done - removed the styling because scoped css doesn't work